### PR TITLE
[styles] Add Bowling Alley

### PR DESCRIPTION
**Pull request by endim8**
icon day

icon night:

(made by me so CC0)

Screenshot 2022-12-24 at 16 52 23

Fixes https://github.com/organicmaps/organicmaps/issues/4144

### CODE REVIEW

**Comment by vng** [comment]
We already have bowling-m for types sport=9pin, sport=10pin.
I think we can/should combine all this types in one icon, no?

**Comment by endim8** [comment]
IMO the old icon doesn't look very good:


the latest push removes the old icons & drawing rules, but i can switch to the old one if wanted :)

**Comment by vng** [comment]
- leisure|bowling_alley;255; is ok, but please, put it somewhere below 255. According to the usage, please keep 255 for more "often" type.
- Well, sport=9pin/10pin maybe a _secondary_ type, but why not to draw specific icon? Like we do for football/basketball/etc ... Other question, is it ok with current POI icon? I think its ok. Or worth to revert back black/white bawling icon?

**Comment by vng** [comment]
@j13m126 
https://github.com/organicmaps/organicmaps/pull/2960

**Comment by endim8** [comment]
> leisure|bowling_alley;255; is ok, but please, put it somewhere below 255.

Will move it lower 👍
> sport=9pin/10pin maybe a secondary type, but why not to draw specific icon? Like we do for football/basketball/etc

it seems there are three ways bowling is mapped:
- as a primary feature eg. https://www.osm.org/node/4767656648
- as a secondary feature (only represented as tags) eg. https://www.osm.org/node/4588495941
- as a secondary feature (as it's own node) eg. https://www.osm.org/node/2234544743

In the first case, the new icon is good.
In the second case, the primary feature icon (pub, leisure_centre, restaurant) should be displayed instead of the bowling icon
in the third case, maybe using the old black and white icon is good, but I'm not sure how useful it would be.

File: data/strings/types_strings.txt [hunk 1]
```diff
@@ -1118,6 +1118,46 @@
...
+    ja = ボーリング場
+    ko = 볼링장
+    mr = गोलंदाजी गल्ली
+    nb = Bowlinghall
+    nl = Bowlingbaan
+    pl = Kręgielnia
+    pt = Pista de boliche
+    pt-BR = Pista de boliche
+    ro = Popicărie
+    ru = Кегельбан
```
Comment (vng) [inline data/strings/types_strings.txt:1150]: ru = Боулинг

File: data/categories.txt [hunk 2]
```diff
@@ -4185,6 +4185,46 @@ sk:3Kino
...
+ja:ボーリング場
+ko:볼링장
+mr:गोलंदाजी गल्ली
+nb:Bowlinghall
+nl:Bowlingbaan
+pl:Kręgielnia
+pt:Pista de boliche
+pt-BR:Pista de boliche
+ro:Popicărie
+ru:Кегельбан
```
Comment (vng) [inline data/categories.txt:4217]: ru:4Боулинг

**Review by vng — COMMENTED** [review]
Ok, I vote for revert back sport=9/10pin black/white icons.
For leisure - use separate new (POI style) icon.

**Comment by vng** [comment]
And yes, we should check draw priorities (e g 9pin icon should NOT replace restaurant or leisure icon). I can add tests later.

**Comment by endim8** [comment]
Screenshot_20230105-150015_Organic Maps


https://www.osm.org/way/28883091

The sport=9/10pin type seems to have a higher priority than bowling alley (this poi has 'leisure=bowling_alley' & 'sport=10pin')

**Review by jimcarst — APPROVED** [review]

**Review by MetehanOzyurek — APPROVED** [review]

**Review by csidirop — APPROVED** [review]

**Comment by endim8** [comment]
fixed type priority
signal-2023-01-05-163757_005

and added correct ru translations 👍

**Comment by endim8** [comment]
finally good to merge :)

File: data/categories.txt [hunk 3]
```diff
@@ -4185,6 +4185,46 @@ sk:3Kino
...
+hu:Tekepálya
+id:Arena Bowling
+it:Sala da bowling
+ja:ボーリング場
+ko:볼링장
+mr:गोलंदाजी गल्ली
+nb:Bowlinghall
+nl:Bowlingbaan
+pl:Kręgielnia
+pt:Pista de boliche
```
Comment (matheusgomesms) [inline data/categories.txt:4214]: pt:Pista de bowling

Comment (endim8) [reply data/categories.txt:4214]: Should `pt-BR:Pista de boliche` be changed to bowling as well?

Comment (matheusgomesms) [reply data/categories.txt:4214]: No, just PT. PT-BR is already correct (Pista de boliche), thanks!

Comment (endim8) [reply data/categories.txt]: cool, thank you for the translations :)

**Review by matheusgomesms — APPROVED** [review]

File: data/strings/types_strings.txt [hunk 4]
```diff
@@ -1118,6 +1118,46 @@
...
+    cs = Bowlingová dráha
+    da = Bowling bane
+    de = Bowlingbahn
+    el = Αίθουσα σφαιρίσεως
+    es = Bolera
+    et = Bowlingusaal
+    eu = Bolatokia
+    fa = ﮓﻨﯿﻟﻮﺑ ﻦﻟﺎﺳ
+    fi = Keilarata
+    fr = Allée de bowling
```
Comment (Kazing) [inline data/strings/types_strings.txt:1136]: fr:Salle de bowling

**Review by Kazing — CHANGES_REQUESTED** [review]
A little change in fr

**Review by LaoshuBaby — APPROVED** [review]

**Comment by endim8** [comment]
> A little change in fr

thank you :) updated

**Review by Kazing — APPROVED** [review]

**Review by lunarna-gh — APPROVED** [review]

**Review by vng — APPROVED** [review]

**Pull request merged**